requirements.txt

```diff
@@ -4,7 +4,7 @@ cattrs==1.1.1
 jsonpickle==1.2
 networkx
 tensorflow>=2.6.3,<2.9.0
-h5py>=3.1.0,<=3.6.0
+h5py==3.1.0
```

Comment:
Let's keep the version range in `requirements.txt` broad -- having a single version or restrictive version ranges will break imports across different platforms. (For example, Colab or systems with different versions of TensorFlow which are pinned to specific h5py versions.)